config/filter.d/nginx-env.conf

```diff
@@ -0,0 +1,13 @@
+# fail2ban filter configuration for nginx 
+# searches for any requests for .env in nginx logs
+
+[Definition]
+failregex = ^<HOST>(?::\d+)?\s+.*.env .*
```

Comment:
Hmm...
this RE looks a bit vulnerable to me (too many catch-alls), let alone dot-char in `.env` matches any character, so I guess it's not really what you want.